hudi-client/hudi-flink-client/src/main/java/org/apache/hudi/io/storage/row/HoodieRowDataCreateHandle.java

```diff
@@ -142,7 +143,11 @@ public void write(String recordKey, String partitionPath, RowData record) throws
             ? HoodieRecordDelegate.create(recordKey, partitionPath, null, newRecordLocation) : null;
         writeStatus.markSuccess(recordDelegate, Option.empty());
       } catch (Throwable t) {
+        LOG.error("Failed to write : key is " + recordKey + ", data is " + rowData, t);
         writeStatus.markFailure(recordKey, partitionPath, t);
```

Comment:
what about the other write handles?

Reply:
like `HoodieCreateHandle` the same logic contains in `doWrite` function.

Reply:
Thanks.  add same logic according to the inheritance relationship of HoodleWriteHandle.